dipy/testing/decorators.py

```diff
@@ -73,3 +73,18 @@ def test_with_xvfb(*args, **kwargs):
     # Plant it back in and return the new function:
     test_with_xvfb.__name__ = fname
     return test_with_xvfb if not is_windows else my_test
+
+
+def set_random_seed(func):
```

Comment:
I would rename this to something like `fix_random_number_generator`, as the seed is fixed, and cannot be fixed from the outside. The name should then be made extensive to the inner method.